cli/operations.py

```diff
@@ -54,6 +54,35 @@ def get_azure_storage_client(config):
     return azure_storage_client
 
 
+def onboard(config, folder_name):
+    blob_storage = get_azure_storage_client(config)
+    uri = 'https://' + config.get("storage_account") + '.blob.core.windows.net/' + config.get("storage_container") + '/'
+    functions_url = config.get('url') + '/api/onboarding'
+    images = []
+    count = 0
+    for image in os.listdir(folder_name):
+        if image.lower().endswith('.png') or image.lower().endswith('jpg') or image.lower().endswith('jpeg'):
```

Comment:
@samiyaakhtar How do we know that we only support these file types? Does VOTT say this? Also why a dot for PNG and not for JPG or JPEG?

Reply:
Updated it to support these